src/wavesurfer.js

```diff
@@ -963,7 +964,7 @@ export default class WaveSurfer extends util.Observer {
         }
 
         this.params.scrollParent = oldScrollParent;
-        this.fireEvent('seek', progress);
+        this.fireEvent('seek', {from:currentTime, to:progress * this.getDuration()});
```

Comment:
This creates 2 breaking changes..
First is the object instead of number - can we pass `from` as second argument?
Second - fraction is transformed to seconds. Shouldn't we stick to fraction between 0 and 1 as unified format for events?
On other hand, if we want to use seconds in event, we should accept seconds in `seekTo` method as well
```
seekTo(amount: number, type?: 'seconds' | 'fraction'): void;
```

Reply:
Exactly, `progress` should remain the first argument. `from` and `to` should come next. They don't even have to be an object, could be just positional arguments.
@entonbiba also be wary that `progress * this.getDuration()` may yield many digits after the comma. You might want to round it.